src/tooltip/tooltip.js

```diff
@@ -315,6 +317,11 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position', 'ui.bootstrap.s
                 $timeout.cancel(transitionTimeout);
                 transitionTimeout = null;
               }
+              
+              if (positionTimeout) {
```

Comment:
Actually. Thought about this more on the way home. I suspect this condition could never occur, and there are no reports of it happening. Don't change what isn't broke

I'll update this tomorrow and remove this